extensions/wikia/WikiaMaps/js/WikiaMapsCreateMapPreview.js

```diff
@@ -237,8 +235,7 @@ define(
 		function trackMapCreation(tileSetData) {
 			var tileSetId = tileSetData.tileSetId,
 				mapTypeChosen = tileSetData.type,
-				label = mapTypeChosen + '-map-created' +
-					((!tileSetId && mapTypeChosen !== 'geo') ? '-with-new-tileset' : '');
+				label = mapTypeChosen + '-map-created' + (!tileSetId ? '-with-new-tileset' : '');
```

Comment:
Do we need to do this concatenation here if there is only one type? What about removing `type` from `tileSetData` entirely?